### PR TITLE
Edit copy, banners and links for Free Month of FUn

### PR DESCRIPTION
**Pull request by mattcleary55**
The changes in this document have been implemented: https://docs.google.com/document/d/1J1rKSZwxJwpuLqopWGBx2SUOrm46yaSgT8BdV-YT1Do/edit

### CODE REVIEW

File: src/pages/commercial.md [hunk 1]
```diff
@@ -102,12 +102,20 @@ testimonial:
...
 renewalBanner:
-  hidden: false
+  hidden: true
   image: /images/uploads/renewal-drone.jpg
   mainText: Read our PfCO application tips.
   buttonText: TAKE ME TO THE POST
   buttonUrl: https://blog.flockcover.com/flocks-essential-guide-to-pfco-renewals-dac39542e16a
   buttonTrack: ''
+promotionBanner:
+  hidden: false
```
Comment (jbreckmckye) [inline src/pages/commercial.md]: Is this supposed to be false?

File: src/templates/home.js [hunk 2]
```diff
@@ -97,42 +105,49 @@ const HomeTemplate = ({
...
-    backgroundSize: '45rem',
-    backgroundRepeat: 'no-repeat',
-    backgroundPosition: 'bottom left',
-    width: '100%',
-    [`@media (min-width: ${R.nth(0, breakpoints)})`]: {
-      backgroundImage: `url(${bigFlock})`,
-      backgroundPosition: 'bottom right'
-    }
+    background: colors.yellow
+    // backgroundImage: `url(${mobileFlock})`,
```
Comment (jbreckmckye) [inline src/templates/home.js]: Are you keeping this to make it easier to undo the changes?

File: src/pages/commercial.md (hunk 1 above)
Comment (mattcleary55) [reply src/pages/commercial.md]: Yup

File: src/templates/home.js (hunk 2 above)
Comment (mattcleary55) [reply src/templates/home.js]: yup!
